plugins/modules/homebrew_cask.py

```diff
@@ -534,7 +534,12 @@ def _upgrade_all(self):
             rc, out, err = self.module.run_command(cmd)
 
         if rc == 0:
-            if re.search(r'==> No Casks to upgrade', out.strip(), re.IGNORECASE):
+            # 'brew upgrade --cask' does not output anything if no casks are upgraded
+            if not out:
```

Comment:
Maybe better add `strip` here so in case it suddenly starts having a newline or some space, it won't stop working:
```suggestion
            if not out.strip():
```

Reply:
LGTM. Passed local tests.